### PR TITLE
Adds the contractors uniforms

### PR DESCRIPTION
**Pull request by Alberyk**
At the request of the lore team, this pr adds a couple of contract related clothing. It also fixes and tweaks some options names and descriptions.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -239,4 +259,19 @@
...
+	path = /obj/item/clothing/under/rank/hephaestus
+
+/datum/gear/uniform/contractors/New()
+	..()
+	var/uniform = list()
+	uniform["hephaestus industries uniform"] = /obj/item/clothing/under/rank/hephaestus
+	uniform["necropolis industries uniform"] = /obj/item/clothing/under/rank/necropolis
+	uniform["idris incorporated uniform"] = /obj/item/clothing/under/rank/idris
+	uniform["einstein engines uniform"] = /obj/item/clothing/under/rank/einstein_engines
+	uniform["zheng-hu uniform uniform"] = /obj/item/clothing/under/rank/zheng
```
Comment (VTCobaltblood) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]: Zeng

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 2]
```diff
@@ -239,4 +259,19 @@
...
+	path = /obj/item/clothing/under/rank/hephaestus
+
+/datum/gear/uniform/contractors/New()
+	..()
+	var/uniform = list()
+	uniform["Hephaestus Industries uniform"] = /obj/item/clothing/under/rank/hephaestus
+	uniform["Necropolis Industries uniform"] = /obj/item/clothing/under/rank/necropolis
+	uniform["Idris Incorporated uniform"] = /obj/item/clothing/under/rank/idris
+	uniform["Einstein Engines uniform"] = /obj/item/clothing/under/rank/einstein_engines
+	uniform["Zheng-hu  uniform"] = /obj/item/clothing/under/rank/zheng
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]: should be Zheng-Hu Pharmaceuticals

File: code/modules/clothing/under/jobs/civilian.dm [hunk 3]
```diff
@@ -173,3 +173,33 @@
...
+	worn_state = "idris"
+
+/obj/item/clothing/under/rank/einstein_engines
+	name = "Einstein Engines uniform"
+	desc = "A uniform worn by Einstein Engines Incorporated employees and contractors."
+	icon_state = "einstein_engine"
+	worn_state = "einstein_engine"
+
+/obj/item/clothing/under/rank/zheng
+	name = "Zheng-hu uniform"
```
Comment (alsoandanswer) [inline code/modules/clothing/under/jobs/civilian.dm]: change here too

File: code/modules/clothing/under/jobs/civilian.dm [hunk 4]
```diff
@@ -173,3 +173,33 @@
...
 	worn_state = "miner"
+
+/obj/item/clothing/under/rank/hephaestus
+	name = "Hephaestus industries uniform"
+	desc = "A uniform worn by Hephaestus Industries employees and contractors."
+	icon_state = "heph"
+	worn_state = "heph"
+
+/obj/item/clothing/under/rank/necropolis
+	name = "Cecropolis Industries uniform"
```
Comment (alsoandanswer) [inline code/modules/clothing/under/jobs/civilian.dm]: necropolis typo

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 5]
```diff
@@ -169,8 +173,23 @@
...
+/datum/gear/uniform/security_contractors
+	display_name = "security contractor uniform selection"
+	description = "A selection of security contractor related uniforms."
+	path = /obj/item/clothing/under/rank/security/necropolis
+	allowed_roles = list("Security Officer", "Warden", "Head of Security","Detective", "Forensic Technician", "Security Cadet")
+
+/datum/gear/uniform/security_contractors/New()
+	..()
+	var/uniform = list()
+	uniform["Necropolis security uniform"] = /obj/item/clothing/under/rank/security/necropolis
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]: here should be the full corporation name
Necropolis Industries

same with the two below as well

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 6]
```diff
@@ -204,6 +205,7 @@
 
 /datum/gear/suit/dominia
 	display_name = "dominia great coat selection"
+	description = "A selection of dominian coats."
```
Comment (skull132) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm]:
```suggestion
	description = "A selection of Dominian coats."
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 7]
```diff
@@ -169,8 +173,23 @@
 	path = /obj/item/clothing/under/rank/head_of_security/navyblue
 	allowed_roles = list("Head of Security")
 
+/datum/gear/uniform/security_contractors
+	display_name = "security contractor uniform selection"
+	description = "A selection of security contractor related uniforms."
```
Comment (skull132) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]:
```suggestion
	description = "A selection of security contractor uniforms."
```

`related` doesn't make sense in this context.

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 8]
```diff
@@ -239,4 +259,19 @@
 
 /datum/gear/uniform/miscellaneous/greensuit
 	display_name = "green formal uniform"
-	path = /obj/item/clothing/under/gov
\ No newline at end of file
+	path = /obj/item/clothing/under/gov
+
+/datum/gear/uniform/contractors
+	display_name = "contractor uniform selection"
+	description = "A selection of contractor related uniforms."
```
Comment (skull132) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm]:
```suggestion
	description = "A selection of contractor uniforms."
```

File: code/modules/clothing/suits/armor.dm [hunk 9]
```diff
@@ -525,6 +525,12 @@
 	body_parts_covered = UPPER_TORSO | LOWER_TORSO | LEGS | ARMS
 	armor = list(melee = 50, bullet = 30, laser = 30, energy = 15, bomb = 40, bio = 0, rad = 0)
 
+/obj/item/clothing/suit/armor/vest/idris
+	name = "Idris reclamation unit coat "
```
Comment (skull132) [inline code/modules/clothing/suits/armor.dm]: Make the capitilization of `reclamation unit` consistent. It's either a proper name or a noun.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 10]
```diff
@@ -173,3 +173,33 @@
 	icon_state = "miner"
 	item_state = "lb_suit"
 	worn_state = "miner"
+
+/obj/item/clothing/under/rank/hephaestus
+	name = "Hephaestus industries uniform"
```
Comment (skull132) [inline code/modules/clothing/under/jobs/civilian.dm]:
```suggestion
	name = "Hephaestus Industries uniform"
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 11]
```diff
@@ -173,3 +173,33 @@
...
+	worn_state = "idris"
+
+/obj/item/clothing/under/rank/einstein_engines
+	name = "Einstein Engines uniform"
+	desc = "A uniform worn by Einstein Engines Incorporated employees and contractors."
+	icon_state = "einstein_engine"
+	worn_state = "einstein_engine"
+
+/obj/item/clothing/under/rank/zheng
+	name = "Zheng-hu uniform"
```
Comment (skull132) [inline code/modules/clothing/under/jobs/civilian.dm]:
```suggestion
	name = "Zheng-Hu uniform"
```

consistent capitlization required.

**Pull request merged**